src/Classes/Category.php

```diff
@@ -32,40 +32,24 @@ public static function groupByCategory($modules)
             unset($groupedModules['library']);
             $groupedModules['library'] = $temp;
         }
-        
+
         return $groupedModules;
     }
 
     public static function getCategoryName($category)
     {
-        if ($category == 'import/export') {
-            return 'Import/Export';
-
-        } elseif ($category == 'persistance') {
-            return 'Datenbank Module';
-
-        } elseif ($category == 'productivity') {
-            return 'Produktivität';
-
-        } elseif ($category == 'promotion/marketing') {
-            return 'Promotion & Marketing';
-
-        } elseif ($category == 'productinfos') {
-            return 'Zusatzinformationen & Produkt-Tabs';
-
-        } elseif ($category == 'shipping') {
-            return 'Versand Module';
-
-        } elseif ($category == 'library') {
-            return 'Programmcode Bibliotheken';
-
-        } elseif ($category == 'nocategory') {
-            return 'Sonstige Module';
-
-        } elseif ($category) {
-            return $category;
-        }
-
-        return 'Sonstige Module';
+        $allCategories = array(
```

Comment:
Looks nice. Can you use 
`['import/export' => 'Import / Export', ... ]`  
instead of 
`array('import/export' => 'Import / Export', ...)`